### DISCUSSION TITLE
multiple monitors/displays in regolith2?

### DISCUSSION THREAD

**Discussion by nrvale0** [regolith-linux/regolith-desktop#700] [General]
Are multiple monitors currently supported with regolith2?

I just tried and:

1) was not able to change the default monitor in Settings
2) I ended up dead space on both screens which manifested as light gray areas unreachable via mouse
3) clicks on one monitor were showing up in apps on the other monitor

This was consistent across reboots.

I also logged in as the "Ubuntu" option in 22.04 and verified that I was able to configure MM as expected via that GUI shell.

This is with Ubuntu 22.04 and the latest Regolith packages as of an update this morning. NVIDIA GA106 [GeForce RTX 3060].

**Comment by binc75** [comment]
Hi @nrvale0,
I've just upgraded my machine to Ubuntu 22.04 and Regolith 2.1 and I'm working with a dual monitor setup through a usb-c docking station. 

![Screenshot from 2022-08-15 11-02-37](https://user-images.githubusercontent.com/48818657/184607455-c1c836d0-ed50-4923-8e69-de70ed80ee67.png)

My machine is a Lenovo X1 Nano.
Cheers
  Nicola

**Comment by nrvale0** [comment]
Hey, Nicola. And everything seems to be working alright?

I can get this far but then very little in the way of further config or even having clicks register on the correct display seems to work. Severely broken on this end. I'm going to have to dig into how one would even debug these situations as MM has always sort of just worked for me and, like I said, this config does seem to work fine if I login to Ubuntu's GNOME-based desktop shell.

**Comment by nrvale0** [comment]
Just looping back on this...

Made another pass at MM last night and got the same results:

- large unreachable gray areas on both screens
- mouse clicks on one display registering in apps on other display
- inability to log back in after logging out due to black screen and no login prompt from display manager
- inability to reposition displays via the Settings app

I played around with xrandr manually to see if I could get things working without using the Settings app and had only marginally more success.

In contrast, everything worked as expected in Ubuntu GNOME session. 

I'm running the latest available proprietary NVIDIA drivers from the System76 apt repos because I had read about (now fixed?) problems with i3 and the Open Source NVIDIA drivers.

**Reply by kgilmer** [reply]
I don't have hardware to reproduce but numerous users have had various display/video related problems due to nvidia hardware across open source and proprietary drivers.  I myself have replaced work laptops that came with nvidia hardware because they're simply not reliable enough.  Other DEs or distro/DE integrations may be doing special things to make the nvidia stuff work :shrug:

**Comment by ethanfel** [comment]
i always had this kind of issue, even with 1.6 and Nvidia 

What i did was too create a xrand script with the settings i want (position resolution etc) and bind it to a shortcut.

xrandr --output HDMI-0 --left-of DP-2
xrandr --output HDMI-0 --rotate right 
xrandr --output DP-2 --mode 3440x1440 --rate 100 --primary

**Reply by nrvale0** [reply]
Timely...

That actually does work now. However, so does configuring via the GUI interface. I'm pretty sure the root resolution is that I installed the NV drivers via the System76 driver packages (.deb). 

Thanks for the advice though.
